ext/default-mail-templates/README.md

```diff
@@ -1,7 +1,7 @@
 # Mail templates
 
-This directory contains the default templates for all mails sent by default as result of activity in
-the marketplace.
+This directory contains the default templates for mails defined in the default transaction process.
+Templates for other notifications can be edited in Console.
```

Comment:
We could stick to same terminology that is used here: https://flex-docs.sharetribe.com/email-templates.html. So built-in emails and transaction emails.